pages/_app.tsx

```diff
@@ -43,6 +43,7 @@ function MyApp({ Component, pageProps }: AppProps) {
                             <div className='container-xl p-responsive height-full my-11'>
                                 <TopActions />
                                 <Component {...pageProps} />
+                                <NotificationManager />
```

Comment:
Setting the manager here makes it impossible to display a notification if the wallet is not connected